lib/data/nextcloud/saber_syncer.dart

```diff
@@ -78,10 +78,10 @@ class SaberSyncInterface
       final bestFile = await getBestFile(syncFile);
       switch (bestFile) {
         case BestFile.local:
-        case BestFile.either:
           // Local file is newer, do nothing
           break;
         case BestFile.remote:
+        case BestFile.either: //local file does not exists
```

Comment:
Why did you change this part? `BestFile.either` should be when the local and remote files are equivalent, so no sync needs to occur for this file

Reply:
BestFile.either is also returned when local file does not exist. I did not searched the code why.

Reply:
Thanks, I've improved this aspect in 01b28d9fb2f62672849de58841c573ebad908443